apps/backend/prisma/schema.prisma

```diff
@@ -67,13 +67,14 @@ model Event {
 }
 
 model DrinkAction {
-  id         String @id @default(uuid())
+  id         String   @id @default(uuid())
   price      Int
   milliliter Int
   drinkId    String
-  drink      Drink  @relation(fields: [drinkId], references: [id])
+  drink      Drink    @relation(fields: [drinkId], references: [id])
   eventId    String
-  event      Event  @relation(fields: [eventId], references: [id])
+  event      Event    @relation(fields: [eventId], references: [id])
   // userId     String
   // user       User   @relation(fields: [userId], references: [authSchId])
+  createdAt  DateTime @default(now())
```

Comment:
also add this to the drink action entity.
But don't remove it from the create drink action dto, the user should be able to create a drink action in the past